internal/admin/command_payment_describe.go

```diff
@@ -31,20 +31,20 @@ func (c *command) describe(cmd *cobra.Command, _ []string) error {
 
 	marketplace := user.GetOrganization().GetMarketplace()
 	if marketplace.GetPartner() != ccloudv1.MarketplacePartner_UNKNOWN {
-		output.Printf("Organization is currently linked to %s Marketplace account.\n", marketplace.GetPartner())
+		color.Printf(c.Config.EnableColor, "Organization is currently linked to %s Marketplace account.\n", marketplace.GetPartner())
```

Comment:
Is there any good way to hide this parameter? Like, what if we make a color or printer struct holding this bool, make the print functions into methods for that struct, and add it to the command structs, so that we could write something like `c.Color.Printf("Organization is...")`.

Reply:
Not all print statements have access to `c` at the moment, but I think they all should be rewritten to... I think after we do that then making this change would be a good idea. In the meantime, let's just have a single set of functions, `output.Print*()` as we discussed in the comment above